internal/pkg/service/stream/keboolasink/bridge/file.go

```diff
@@ -131,18 +131,38 @@ func (b *Bridge) registerFileImporter() {
 
 			// Get file upload credentials
 			var keboolaFile *keboolasink.File
-			err = b.schema.File().ForFile(file.FileKey).GetOrNil(b.client).WithResultTo(&keboolaFile).Do(ctx).Err()
+			prefix := b.schema.File().ForFile(file.FileKey)
+			err = prefix.GetOrNil(b.client).WithResultTo(&keboolaFile).Do(ctx).Err()
 			if err != nil {
 				return err
 			}
 
 			// Authorized API
 			api := b.publicAPI.WithToken(token.TokenString())
 
-			// Create job to import data
-			job, err := api.LoadDataFromFileRequest(keboolaFile.TableKey, keboolaFile.FileKey).Send(ctx)
-			if err != nil {
-				return err
+			// Check if job already exists
+			var job *keboola.StorageJob
+			if keboolaFile.StorageJobID != nil {
+				job, err = api.GetStorageJobRequest(keboola.StorageJobKey{ID: *keboolaFile.StorageJobID}).Send(ctx)
+				if err != nil {
+					return err
+				}
+
+				if job.Status == "success" {
```

Comment:
It would be nice to add constants to go-client.

Reply:
https://github.com/keboola/go-client/pull/139